src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -349,6 +349,13 @@ pub const RLIMIT_UMTXP: ::c_int = 14;
 #[deprecated(since = "0.2.64", note = "Not stable across OS versions")]
 pub const RLIM_NLIMITS: ::rlim_t = 15;
 
+pub const NI_NOFQDN: ::c_int = 0x00000001;
+pub const NI_NUMERICHOST: ::c_int = 0x00000002;
+pub const NI_NAMEREQD: ::c_int = 0x00000004;
+pub const NI_NUMERICSERV: ::c_int = 0x00000008;
+pub const NI_DGRAM: ::c_int = 0x00000010;
+pub const NI_NUMERICSCOPE: ::c_int = 0x00000020;
```

Comment:
This causes CI error on FreeBSD 10 and 11, it seems they don't have?: https://github.com/freebsd/freebsd/commit/43835a137db2c11db762c613be4c1be506f8cfb8
If so, then could you also tweak libc-test? You should touch here:

https://github.com/rust-lang/libc/blob/0dc26a9f290e683fa109a1bbcccced08cc85ee41/libc-test/build.rs#L1733

Reply:
This should be fixed now